backend/core/src/shared/entities/address.entity.ts

```diff
@@ -32,10 +32,9 @@ export class Address {
 
   @Column({ type: "text", nullable: true })
   @Expose()
-  @IsOptional({ groups: [ValidationsGroupsEnum.partners] })
   @IsString({ groups: [ValidationsGroupsEnum.default] })
   @MaxLength(64, { groups: [ValidationsGroupsEnum.default] })
-  city?: string | null
+  city: string | null
```

Comment:
I think `| null` can be removed too if they are not optional because frontend side we do not distinguish between undefined and null, right?